components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/internal/OAuth2ServiceComponent.java

```diff
@@ -146,21 +146,30 @@ protected void setAuthenticationMethodNameTranslator(
     }
 
     @Reference(
-            name = "identity.oauth.par.service.component",
-            service = ParAuthService.class,
-            cardinality = ReferenceCardinality.MANDATORY,
+            name = "request.builder.service",
```

Comment:
Lets change the service name to reflect the changes done for the class. i.e. registering this as an oauth authorization request builder